src/test/java/ua/ithillet/lesson5/ArrayImplementationOfListTest.java

```diff
@@ -0,0 +1,67 @@
+package ua.ithillet.lesson5;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ArrayImplementationOfListTest {
+
+    @Test
+    public void CheckGetAllMethod() {
+        String[] arrayWords = {"have", "be", "bear", "lose", "mean"};
+        var arrayImplementationOfList = new ArrayImplementationOfList(arrayWords);
+        assertTrue(Arrays.equals(arrayImplementationOfList.getAll(), arrayWords));
```

Comment:
```suggestion
        assertEquals(arrayImplementationOfList.getAll(), arrayWords);
```
Різниці в результаті немає (коли все працює). 
Різниця є маленька, якщо тест стає червоним: 
1. assertTrue видає `expected: true, actual: false` 
2. assertEquals видає `expected: [....], actual: [.....]` 
Інколи це може допомогти раніше зрозуміти в чому проблема.